### DISCUSSION TITLE
挂载 SMB/CIFS 网络共享文件夹问题

### DISCUSSION THREAD

**Discussion by baogenwang** [coolsnowwolf/lede#9772] [Bugs / 问题反馈]
x86  518内核 我的U是n5105 设置挂载 SMB/CIFS 网络共享文件夹的时候。重起一下就会失效必需要重新点一下应用才可以，那个大佬可以解决吗？

**Comment by EnnawYang** [comment]
https://github.com/coolsnowwolf/lede/issues/6035 ，你可以试下，反正我后来不需要了没试过
